includes/class-coblocks-form.php

```diff
@@ -961,8 +961,16 @@ public function process_form_submission( $atts ) {
 		 */
 		$email_content = (string) apply_filters( 'coblocks_form_email_content', $this->email_content, $_POST, $post_id );
 
-		$sender_name  = isset( $_POST[ $name_field_id ]['value'] ) ? esc_html( $_POST[ $name_field_id ]['value'] ) : '';
 		$sender_email = isset( $_POST[ $email_field_id ]['value'] ) ? esc_html( $_POST[ $email_field_id ]['value'] ) : '';
+		$sender_name  = isset( $_POST[ $name_field_id ]['value'] ) ? esc_html( $_POST[ $name_field_id ]['value'] ) : '';
+
+		$headers = [];
```

Comment:
@EvanHerman Is WP cool with shorthand array syntax now? (I am)

Reply:
@fjarrett-godaddy You are right - I forgot it's against WP coding standards. Good call.